cmake/Geometry.cmake

```diff
@@ -29,19 +30,28 @@ set(PHOEBUS_ANALYTIC_GEOMETRIES
     "MonopoleSph"
     "MonopoleCart"
     )
+
+# List geometries that should not be cached here
 set(PHOEBUS_GEOMETRY_NO_CACHE
```

Comment:
Is geometries that shouldn't be cached only for geometries currently under development, or should some geometries *never* be cached? Same question for `MUST_CACHE`.

Reply:
My thought process here was that there may be some geometries inappropriate for caching. Original the monopole geometries were like this until I added things like time-dependence.

But for example, a geometry that uses the grid through some mechanism that isn't the cache probably shouldn't be cached.

The `MUST_CACHE` variable was originally for `MonopoleCart` until I figured out I didn't need it. We could remove these two lists until we need them. It would make the cmake code a bit simpler.

Reply:
I've removed the `NO_CACHE` and `MUST_CACHE` lists. Let's keep it neat and clean until we need it.